config/locales/ca.yml

```diff
@@ -345,6 +349,25 @@ ca:
       text: "%{organization_name} en"
       text_donation: L'associació ADBdT ofereix TimeOverflow de forma gratuïta als Bancs de Temps. Amb una donació de %{href} pots ajudar a mantenir i millorar la plataforma.
       text_donation_link: 1€ al mes
+  multi_transfers:
+    multi_transfers: 'Transferències múltiples '
```

Comment:
Do we need that extra space at the end of the line? I see that in many copies.

Reply:
Not need it, i'll check it

Reply:
There is not spaces in localeapp, there is somthing that adds localeapp :thinking: also I'm not sure what rule localeapp use to add quotes...

Reply:
:see_no_evil: